Application/EdFi.Admin.DataAccess/Contexts/IUsersContext.cs

```diff
@@ -16,7 +16,7 @@ public interface IUsersContext : IDisposable
     {
         DbSet<User> Users { get; set; }
 
-        DbSet<ApiClient> Clients { get; set; }
+        DbSet<ApiClient> ApiClients { get; set; }
```

Comment:
Why change this? This will also cause breaking changes in the upcoming Admin API 1 and Admin App releases. I do agree that `ApiClients` is a _better_ name, but I'm hesitant about causing us extra work by changing this interface.